src/Data/Aeson.hs

```diff
@@ -162,6 +163,12 @@ import qualified Data.ByteString.Lazy as L
 encode :: (ToJSON a) => a -> L.ByteString
 encode = encodingToLazyByteString . toEncoding
 
+-- | Efficiently serialize a JSON value as a strict 'B.ByteString'.
```

Comment:
I don't like this. This is not *efficiently*. This is a helper, which perfectly shouldn't be needed, e.g. if you serialise web api response, `wai` takes lazy bytestring.

Reply:
I can change the comment to 

```haskell
-- | Serialize a JSON value as a strict 'B.ByteString'.
```

My use case is not with `wai`, but constructing HTTP client requests for use in tests.

Reply:
Also, convenience functions are very common in libraries and ease the developer burden. `forM_ = flip mapM_`, for example.

Reply:
Adds symmetry to the API as well, `decode`, `decodeStrict` now `encode`, `encodeStrict`